spec/tictactoe_spec.rb

```diff
@@ -7,7 +7,22 @@
   let (:test_ttt) { TicTacToe.new(game_io, player1, player2) }
 
   it "runs the game loop " do
-    allow(test_ttt.game_loop).to receive(:game_loop) { }
-    test_ttt.game_loop
+    expect(test_ttt.game_loop).to receive(:game_loop).once
+    expect(test_ttt).to receive(:play_again?).and_return(false)
+    test_ttt.play_game
+  end
+
+  describe "the play_again? method " do
```

Comment:
I changed `play_again?` to private and made the `new_game` method now is part of the `TicTacToe` class. It is also private. Both are tested through the `play_game` method of `TicTacToe`